src/nvim/vim.h

```diff
@@ -253,7 +253,7 @@ enum { FOLD_TEXT_LEN = 51, };  //!< buffer size for get_foldtext()
 #define STRNCAT(d, s, n)    strncat((char *)(d), (char *)(s), (size_t)(n))
 #define STRLCAT(d, s, n)    xstrlcat((char *)(d), (char *)(s), (size_t)(n))
 
-#define vim_strpbrk(s, cs) (char_u *)strpbrk((char *)(s), (char *)(cs))
+#define VIM_STRPBRK(s, cs) (char_u *)strpbrk((char *)(s), (char *)(cs))
```

Comment:
maybe we can just use `strpbrk` at usage places? most seem to artifically convert _to_ char_u anyway